addon/index.js

```diff
@@ -28,4 +30,36 @@ function decoratorWithReturnValue(fn) {
  * ```
  * @function
  */
-export const next = decoratorWithReturnValue(run.next);
\ No newline at end of file
+export const next = decoratorWithReturnValue(run.next);
+
+
+function decoratorWithDebounce(fn) {
```

Comment:
Did you extract this macro for a reason? I think we could probably use it for `throttle`, just wondering